runtime/vm/JFRChunkWriter.hpp

```diff
@@ -60,6 +60,7 @@ enum MetadataTypeID {
 	ThreadStartID = 2,
 	ThreadEndID = 3,
 	ThreadSleepID = 4,
+    JVMInformationID = 86,
```

Comment:
are you using tabs or spaces? indentation looks off in this file

Reply:
Fixed